app/src/main/java/org/mozilla/fenix/search/awesomebar/AwesomeBarView.kt

```diff
@@ -255,7 +256,7 @@ class AwesomeBarView(
     }
 
     private fun isBrowsingModePrivate(): Boolean {
-        return (container.context.asActivity() as? HomeActivity)?.browsingModeManager?.mode?.isPrivate
+        return (context.asActivity() as? HomeActivity)?.browsingModeManager?.mode?.isPrivate
```

Comment:
would `context.asActivity() as? HomeActivity)?` ever be null?

Reply:
I'm not sure :)